pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -94,6 +97,10 @@ export class ConfigurationCard extends React.Component {
         });
 
         $("#system_information_ssh_keys").on("hide.bs.modal", () => this.host_keys_hide());
+
+        this.realmd.addEventListener("changed", () => {
+            this.setState({});
+        });
```

Comment:
Please no function blocks for simple expressions.

Reply:
Okay. :-)  (I like to use statement blocks for side-effects, and expressions for side-effect free things.)